model/generate.xml

```diff
@@ -100,6 +100,10 @@
     <argument name="RIPEMD160" stdin="true" type="hash160" description="The Base16 hash to convert. If not specified the value is read from STDIN."/>
   </command>
 
+  <command symbol="address-to-key" output="base16" category="WALLET" description="Derive the internal lookup key from an address.">
```

Comment:
“Derive the payments search key of a payment address.”

Reply:
Just double checking on this one:  Not "Derive the payment search key _from_ a payment address"?

Reply:
It was intentional, it is the key of the address. It could reasonably be for or from, but I consider it a property of the address.